### PR TITLE
Returns stabilizer harnesses to their former glory

### LINKED ISSUE #16245: [Bug]: Medbay compression harness have a device power cell instead of a normal cell
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.
- [X] The round I am reporting the bug from, or I am going to talk about, has already ended.

### Location

Horizon

### Describe the issue

After a recent merge all of medbay compression harness spawn with a basic device power cell instead of a normal cell which last absurdly little.

I was told on discord this is a know bug but i decided to make a report simply for the sake of records keeping

### How to reproduce

Start the round and check the harness

### Round ID

N/A

### Staff ckey

N/A

### PR DESCRIPTION
**Pull request by sheeplets**
Fixes #16245 
Replaces those newer 100w device cells with standard 1000w's, hopefully this fixes charge issues.

### CODE REVIEW

**Comment by sheeplets** [comment]
!bugfix

**Comment by Generalcamo** [comment]
Can confirm the bugfix. This was not intentional, and the description of the device cell (both old and new) did not lend itself to this usage.

File: code/game/objects/items/devices/auto_cpr.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 /obj/item/auto_cpr/Initialize()
 	. = ..()
-	battery = new /obj/item/cell/device(src)
+	battery = new /obj/item/cell/(src)
```
Comment (Generalcamo) [inline code/game/objects/items/devices/auto_cpr.dm:44]:
```suggestion
	battery = new /obj/item/cell(src)
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/auto_cpr.dm (hunk 1 above)
Comment (sheeplets) [reply code/game/objects/items/devices/auto_cpr.dm]: businessmen-handshake-handshake

**Review by Alberyk — APPROVED** [review]

**Pull request merged**